packages/components/form/src/form-field.stories.ts

```diff
@@ -0,0 +1,89 @@
+import type { FormField } from './form-field.js';
+import type { Meta, StoryObj } from '@storybook/web-components';
+import '@sl-design-system/text-field/register.js';
+import { type TemplateResult, html } from 'lit';
+import '../register.js';
+
+type Props = Pick<FormField, 'hint' | 'label'> & {
+  customValidity?: string;
+  slot?: TemplateResult;
+};
+type Story = StoryObj<Props>;
+
+export default {
+  title: 'Form/Form Field',
+  argTypes: {
+    slot: {
+      table: {
+        disable: true
+      }
+    }
+  },
+  render: ({ customValidity, hint, label, slot }) => {
+    // Force the text field to report its validity when the story is loaded
+    setTimeout(() => document.querySelector('sl-text-field')?.reportValidity());
+
+    return html`
+      <sl-form-field .hint=${hint} .label=${label}>
+        ${slot ?? html`<sl-text-field .customValidity=${customValidity}></sl-text-field>`}
+      </sl-form-field>
+    `;
+  }
+} satisfies Meta<Props>;
+
+export const Basic: Story = {
+  args: {
+    label: 'Label'
+  }
+};
+
+export const Error: Story = {
+  args: {
+    ...Basic.args,
+    slot: html`<sl-text-field required></sl-text-field>`
+  }
+};
+
+export const Hint: Story = {
+  args: {
+    ...Basic.args,
+    hint: 'This is a hint'
+  }
+};
+
+export const Both: Story = {
+  args: {
+    ...Basic.args,
+    customValidity: 'This is an error',
+    hint: 'This is a hint'
+  }
+};
+
+export const CustomError: Story = {
+  args: {
+    ...Basic.args,
+    slot: html`
+      <sl-text-field required></sl-text-field>
+      <sl-error>This is a <strong>custom</strong> error</sl-error>
+    `
+  }
+};
+
+export const CustomHint: Story = {
+  args: {
+    ...Basic.args,
+    slot: html`
+      <sl-text-field></sl-text-field>
+      <sl-hint>This is a <strong>custom</strong> hint</sl-hint>
+    `
+  }
+};
+
+export const CustomLabel: Story = {
```

Comment:
This is more a question to another custom label example in the storybook with info icon, the tooltip text there connected with the icon is not read by the screen reader?